### ISSUE TITLE
Display Sensory events differently in the graph representation

### ISSUE THREAD

**Issue by davfo224** [ing-bank/baker#110]
Hi
It would be nice if sensory events would be visible in the graph representation of a recipe. You could use bold characters for the event name (it would also work on the representation of a recipe execution).

David

**Comment by merlijn** [comment]
This is a nice idea, thank you :) 

We will try to get this into 1.4.0

**Comment by merlijn** [comment]
I just committed  a change to allow this.

Example (paste in webgraphviz.com):

```
digraph {
	node [fontname = "ING Me", fontsize = 22, fontcolor = white]
	pad = 0.2
	interactionOneOriginalIngredient [shape = circle, color = "#FF6200", style = filled]
	interactionOneRenamed [shape = rect, margin = 0.5, color = "#525199", style = "rounded, filled", penwidth = 2]
	interactionOneRenamedSuccessful -> interactionOneOriginalIngredient
	initialIngredient -> interactionOneRenamed
	interactionOneRenamed -> interactionOneRenamedSuccessful
	interactionOneRenamedSuccessful [shape = diamond, margin = 0.3, style = "rounded, filled", color = "#767676"]
	initialIngredient [shape = circle, color = "#FF6200", style = filled]
	InitialEvent [shape = diamond, margin = 0.3, style = "rounded, filled", fillcolor = "#60A6DA", color = "#525199", penwidth = "12.0"]
	InitialEvent -> initialIngredient
}
```

Would this work, what would you like to see different? 

We will also discuss with the team and together with your feedback can decide on a final look soon.

**Comment by davfo224** [comment]
Looks fine, how about the following for the sensory event:
InitialEvent [shape = diamond, margin = 0.3, style = "rounded, filled", fillcolor = "#**AAAAAA**", fontcolor="#**525199**", color = "#525199", penwidth = "**8.0**"]
But what you propose is also good enough

**Comment by merlijn** [comment]
closing this since it is fixed on master, will be released in 1.4.0
